src/main/paradox/docs/api/assets/realms/realm-replace.json

```diff
@@ -0,0 +1,9 @@
+{
+  "name": "realm1",
+  "openid-configuration": "http://dev.nexus.ocp.bbp.epfl.ch/auth/realms/bbp-test/.well-known/openid-configuration",
```

Comment:
I think we should use `camelCase` for field names, so `openidConfiguration` instead of `openid-configuration` etc.